src/components/pages/account-page/components/group-tab/components/table/student-table/components/EditingColumn.tsx

```diff
@@ -177,7 +185,8 @@ const EditingColumn: FC<EditingColumnProps> = ({ student, refetch }) => {
   }
 
   if (
```

Comment:
rewrite architecture of this if by deleting it. code above should implement permission checking near the buttons where it's used. it will be not duplicated